contrib/grimshot

```diff
@@ -12,13 +12,22 @@
 ##
 ## See `man 1 grimshot` or `grimshot usage` for further details.
 
-getTargetDirectory() {
+get_target_directory() {
   test -f "${XDG_CONFIG_HOME:-$HOME/.config}/user-dirs.dirs" && \
     . "${XDG_CONFIG_HOME:-$HOME/.config}/user-dirs.dirs"
 
   echo "${XDG_SCREENSHOTS_DIR:-${XDG_PICTURES_DIR:-$HOME}}"
 }
 
+get_target_file() {
+  NAME=$(date +%Y%m%d_%H%M%S)
+  DIR=${1:-$(get_target_directory)}
+  while [ -e "$DIR/$NAME$SUFFIX.png" ]; do
+    SUFFIX="_$((i+=1))"
+  done
+  echo "$DIR/$NAME$SUFFIX.png"
+}
```

Comment:
This might be raceful if somebody is taking lots of screenshots in succession (e.g.: when trying to catch a quirk in an animation or something alike). Two instances of grimshot will run, detect no file, and then both write (at the same time) into the new filename.

I'm not saying we should block this merge due to this issue, but mostly pointing it out. I'm not currently using `grimshot`, so have no strong opinion.